app/src/main/java/com/bootcamp_android/parking_app/adapter/DataModel.kt

```diff
@@ -0,0 +1,14 @@
+package com.bootcamp_android.parking_app.adapter
+
+sealed class DataModel {
```

Comment:
What is the intention of using a sealed class. to hold model data classes?

Reply:
I thought of making the classes inherit from a sealed model that cannot be instantiated so that any class that inherits can be used in the adapter, the other option is an interface, what do you think?

Reply:
The disadvantage of using a sealed class to hold your datamodels, is that your models can grow a lot (if the projects gets more complex by time goes by), so your sealed DataModel class can be huge.
You can also use inheritance to do what you want to achieve.
It's great you are trying to use sealed class, you could use _sealed_ for example when you want to handle errors, there are lot's of kind of errors and those can be sealed. Try to find for more examples. 
Keep learning Fer, you are doing great.